form-link/src/main/java/com/ritense/formlink/service/impl/CamundaFormAssociationService.java

```diff
@@ -90,6 +91,18 @@ public CamundaFormAssociationService(
         this.formFieldDataResolvers = formFieldDataResolvers;
     }
 
+    private static <T> Collector<T, ?, Optional<T>> singleElementCollector() {
```

Comment:
Can't this whole collector be replaced by using `.findFirst()` on the stream?

Reply:
don't know what code is this but i didn't write it. for some reason it appears as i did it. but i didn't.